configuration/configuration.go

```diff
@@ -48,6 +48,11 @@ type Configuration struct {
 	// deprecated. Please use Log.Level in the future.
 	Loglevel Loglevel `yaml:"loglevel,omitempty"`
 
+	// DriverPlugins is a path where driver plugins are expected to be found
+	DriverPlugins *string `yaml:"driverplugins,omitempty"`
```

Comment:
Does it mean we should load all the plugins on start? Or scan for them on demand?
For example, if there are some paths like 
```yaml
 - /plugin/a.so
-  /plugin/b.so
```
does distribution load all of them on startup before looking up for a builtin plugin?